packages/light-client-experimental/scripts/testing.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  Cause,
+  Duration,
+  Effect,
+  Logger,
+  LogLevel,
+  pipe as $,
```

Comment:
Is this an Effect convention? Offhand I found $ confusing when looking below because it's been so overloaded in the past in JS libs!

Reply:
Nope, usually you'd do `some_effect.pipe(...whatever)`, but prettier make it look ugly as hell, so i prefer this